server/src/main/java/com/cloud/server/ManagementServerImpl.java

```diff
@@ -2820,6 +2821,7 @@ public long getMemoryOrCpuCapacityByHost(final Long hostId, final short capacity
         cmdList.add(CreateSnapshotCmd.class);
         cmdList.add(CreateSnapshotFromVMSnapshotCmd.class);
         cmdList.add(DeleteSnapshotCmd.class);
+        cmdList.add(ArchiveSnapshotCmd.class);
```

Comment:
@syed I believe you forgot to commit `ArchiveSnapshotCmd` itself!

Reply:
@khos2ow You're right. I'll add it now